src/ingest_validation_tools/directory-schemas/confocal-v2.yaml

```diff
@@ -1,4 +1,95 @@
-draft: true
 files:
   -
-    draft_link: 'https://docs.google.com/spreadsheets/d/1KLY5iXZbbb_5RNR_lZgGSqHiAKBbe34YrtrywSuUC3M'
\ No newline at end of file
+    pattern: extras\/.*
+    required: True
+    description: Folder for general lab-specific files related to the dataset. [Exists in all assays]
+  -
+    pattern: extras\/microscope_hardware\.json
+    required: True
+    description: 
```

Comment:
This is causing errors when generating docs

Reply:
I added the description for microscope_hardware.json